vtm-ios/src/org/oscim/ios/IOSMapApp.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2013 Hannes Janetzek
+ *
+ * This file is part of the OpenScienceMap project (http://www.opensciencemap.org).
+ *
+ * This program is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.oscim.ios;
+
+import com.badlogic.gdx.utils.SharedLibraryLoader;
+import org.oscim.backend.GLAdapter;
+import org.oscim.gdx.GdxAssets;
+import org.oscim.gdx.GdxMap;
+import org.oscim.ios.backend.IosGL;
+import org.oscim.ios.backend.IosGraphics;
+import org.oscim.tiling.TileSource;
+import org.oscim.tiling.source.oscimap4.OSciMap4TileSource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IOSMapApp extends GdxMap {
+
+    public static final Logger log = LoggerFactory.getLogger(IOSMapApp.class);
+
+    public static void init() {
+        // load native library
+        new SharedLibraryLoader().load("vtm-jni");
+        // init globals
+		IosGraphics.init();
+		GdxAssets.init("assets/");

```

Comment:
I also have seen issues with desktop gdx depending on backend and platform, see #2 and #5.
Eventually I had to use the [GLAdapter.GDX_DESKTOP_QUIRKS ](https://github.com/mapsforge/vtm/blob/master/vtm/src/org/oscim/backend/GLAdapter.java#L27) to provide alternartive handling. If there is need we could add a flag for iOS too.